### ISSUE TITLE
${OPENSHIFT_REPO_DIR} is empty after deployment.

### ISSUE THREAD

**Issue by killermenpl** [Skamander/openshift-scaled-play2-cartridge#1]
Once you deploy (using `rhc deploy`), the repo dir is empty, which means there is no bin/ folder and control script fails.
To fix it simply replace 
`APP_DIR="${OPENSHIFT_REPO_DIR}"`
with
`APP_DIR="${OPENSHIFT_HOMEDIR}app-root/runtime/play-openshift/"`
which is exactly the way it used to be before your lates commit. I did it in my fork and now it works perfectly.

**Comment by Skamander** [comment]
Hmm, for me this works: http://i.imgur.com/bTHBA14.jpg?1 Could you tell me the steps you took?

**Comment by killermenpl** [comment]
I simply run the app creation script. I tried both with my repo and yours. Also tried running your script, without any modifications. It just didn't work for me. Control script kept crashing on
'ls $APP_BIN_DIR/ | head -n 1' with 'No such file' error. I did ssh to my app and the repo dir was empty. Changing from ${openshift_repo_dir} to ${openshift_app_dir} fixed that for me and now it works perfectly fine with no problems.

**Comment by Skamander** [comment]
Hmm, I can't reproduce this issue (yet). Tried it also in a Ubuntu VM and it worked. The only way to get a simmilar error (or is it the same?) was when I didn't add `topLevelDirectory := Some("repo")` into the `build.sbt`, or if I changed the name from `repo` to something else.

Then I get the following error:

```bash
starting play...
ls: cannot access /var/lib/openshift/theUUIDofTheApp/app-root/runtime/repo/bin: No such file or directory
```

But I will keep trying to reproduce this. :)
